spa/src/components/portal/ContributionsList/ContributionDetail/ContributionDetail.test.tsx

```diff
@@ -184,6 +199,32 @@ describe('ContributionDetail', () => {
       expect(screen.getByTestId('mock-billing-details').dataset.editable).toBe('false');
     });
 
+    describe('PLUS plan', () => {
+      it("BillingDetails doesn't allow edit mode if contribution.is_modifiable = true", () => {
```

Comment:
Think it _does_ allow edit mode in this case?

Reply:
That's correct. My bad
```suggestion
      it("BillingDetails allows edit mode if contribution.is_modifiable = true", () => {
```